### ISSUE TITLE
Feature request Inline Edit

### ISSUE THREAD

**Issue by prashantnirgun** [eboominathan/Simple-server-side-Datatables-in-Codeigniter#1]
Hi

This is Prashant here I hope you remember me. Unfortunately I deleted my facebook account on data leaks. Just quick ref We had talk on Vue.js + CodeIgniter I hope you remember it.

I am searching for the Inline edit with datatable. I got plugin for Cell edit which is for every column it ask for confirm that is too difficult I need for all columns in table we should have edit option for some column we may have select2, dropdown calendar, Option for that we are maintaing header array which has column details like visible, invisible, index, widgets like select2.

Can we do it.

**Comment by eboominathan** [comment]
yes you can do it! check this example provided by official datatable site
https://editor.datatables.net/examples/inline-editing/simple

**Comment by prashantnirgun** [comment]
Its paid version. Editor has no plugin / Extension for edit its bundled in editor
